tcl/envmngt.tcl.in

```diff
@@ -1236,6 +1278,25 @@ ml <- function(...){
 }}
          }
       }
+      pwsh {
+         if {[getState is_win]} {
+            reportErrorAndExit {No autoinit mode available for 'pwsh' shell on Windows}
+         }
+
+         set fdef "function global:envmodule {"
+         append fdef {
```

Comment:
Support for `MODULES_REDIRECT_OUTPUT` should be added (output sent to stderr or stdout)